### PR TITLE
[bitnami/redis-cluster] Add goss tests

### PR DESCRIPTION
**Pull request by FraPazGal**
Signed-off-by: FraPazGal <fdepaz@vmware.com>

**Description of the change**

Adds Goss tests to the `bitnami/redis-cluster` chart.

**Benefits**

Ensuring higher quality of the Helm chart.

**Possible drawbacks**

Automated tests could show to be potentially flaky. 

**Test results** - PASS

[Please see them in the history here](https://github.com/FraPazGal/charts/pull/8)

**Checklist**

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by superaleks — DISMISSED** [review]
LGTM!

File: .vib/redis-cluster/goss/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+command:
+  redis-version:
+    exec: redis-server --version | grep --quiet "$APP_VERSION"
+    exit-status: 0
+    stdout: [ ]
+    stderr: [ ]
+    timeout: 20000
+  redis-auth:
+    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h 127.0.0.1 -p 6379
```
Comment (joancafom) [inline .vib/redis-cluster/goss/goss.yaml:9]: Use a dynamic port instead, based on the environment variable that configures it:
```suggestion
    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h 127.0.0.1 -p $REDIS_PORT
```

Comment (joancafom) [inline .vib/redis-cluster/goss/goss.yaml:9]: Either use a dynamic port instead, based on the environment variable that configures it:
```suggestion
    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h 127.0.0.1 -p $REDIS_PORT
```

or just suppress the `-h` and/or `-p` part and use the default (`127.0.0.1` & `6379`)

Comment (FraPazGal) [reply .vib/redis-cluster/goss/goss.yaml:9]: I'll follow your 2nd suggestion and remove those flags as I'm already using the default values.

Comment (superaleks) [reply .vib/redis-cluster/goss/goss.yaml]: This is something we could do in the original redis tests.

**Review by joancafom — APPROVED** [review]
LGTM

**Pull request merged**